test_spack.py

```diff
@@ -228,6 +228,9 @@ class IconToolsTest(unittest.TestCase):
     depends_on = {'eccodes', 'cosmo-grib-api'}
     machines = all_machines
 
+    def test_install(self):
```

Comment:
Please add a reason why this is tested or where this is used.
I think in this case it can be as simple as: "So C2SM can use icontools."